src/util.cpp

```diff
@@ -592,26 +592,41 @@ fs::path GetConfigFile(const std::string& confPath)
     return pathConfigFile;
 }
 
+void ArgsManager::ReadConfigStream(fs::ifstream& streamConfig)
```

Comment:
I named if for the fact it got an`ifstream` as argument, but you're right that it's probably better to just name it `ReadConfigFile`.